### PR TITLE
Clarify session requirement for conversion calculations

### PR DESCRIPTION
**Pull request by tyleryle**
### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/product_analytics/journeys/_index.md [hunk 1]
```diff
@@ -31,11 +31,11 @@ With [funnel analysis][1], you can understand the end-to-end conversion of a sin
 
 {{< img src="/product_analytics/journeys/pa-conversions.png" alt="View conversion rate values over time in graphs ">}}
 
-You can view [conversion rates][3] over time by showing the conversion rate value in graphs.
+You can view [conversion rates][3] over time by showing the conversion rate value in graphs. Note that only completed RUM sessions contribute to conversion rates.
```
Comment (estherk15) [inline content/en/product_analytics/journeys/_index.md:34]:
```suggestion
You can view [conversion rates][3] over time by showing the conversion rate value in graphs. **Note**: Only completed RUM sessions contribute to conversion rates.
```

File: content/en/product_analytics/journeys/_index.md [hunk 2]
```diff
@@ -31,11 +31,11 @@ With [funnel analysis][1], you can understand the end-to-end conversion of a sin
...
 
 ## Further reading
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: /product_analytics/journeys/funnel_analysis
 [2]: /product_analytics/journeys/sankey
-[3]: https://app.datadoghq.com/product-analytics/user-journey/conversion?
\ No newline at end of file
+[3]: https://app.datadoghq.com/product-analytics/user-journey/conversion?
```
Comment (estherk15) [inline content/en/product_analytics/journeys/_index.md:41]: This redirects to `https://app.datadoghq.com/product-analytics/user-journey/funnel?`. Is that expected?

Comment (tyleryle) [reply content/en/product_analytics/journeys/_index.md:41]: This was the original relative link to take users to their internal platform. I believe that is intended.

**Review by estherk15 — APPROVED** [review]

**Pull request merged**